.github/workflows/nightly.yml

```diff
@@ -31,31 +31,241 @@ jobs:
           mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=8 -Dprofile=full -DjavaVersion=8 -DoutputDirectory=app -Dgoals="clean package"
           rm -rf app
 
+      - name: Run Archetype for EE 8, SE 8, with Docker
```

Comment:
Since we know the Dockerfile is really not generated based on runtime and not the other parameters, I think we can reduce the number of tests a little. I think lets do the Dockerfile check only once for the very first one for a runtime - so probably just keep this one and not the rest of it's kind.